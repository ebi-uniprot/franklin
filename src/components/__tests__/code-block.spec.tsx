import { render, screen } from '@testing-library/react';

import CodeBlock from '../code-block';

describe('DoughnutChart component', () => {
  test('should render in dark mode', () => {
    const { asFragment } = render(<CodeBlock>Some code</CodeBlock>);
    // TODO: this should be a visual snapshot eventually
    expect(asFragment()).toMatchSnapshot();
  });

  test('should render in light mode', () => {
    const { asFragment } = render(<CodeBlock lightMode>Some code</CodeBlock>);
    // TODO: this should be a visual snapshot eventually
    expect(asFragment()).toMatchSnapshot();
  });
});
