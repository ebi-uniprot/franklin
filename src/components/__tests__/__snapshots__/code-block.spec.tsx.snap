// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DoughnutChart component should render in dark mode 1`] = `
<DocumentFragment>
  <pre
    class="codeblock"
  >
    <code>
      Some code
    </code>
  </pre>
</DocumentFragment>
`;

exports[`DoughnutChart component should render in light mode 1`] = `
<DocumentFragment>
  <pre
    class="codeblock codeblock-light"
  >
    <code>
      Some code
    </code>
  </pre>
</DocumentFragment>
`;
